### ISSUE TITLE
Does not work with Ember Fastboot

### ISSUE THREAD

**Issue by devotox** [dunnkers/ember-polymer#56]
This should have a way of not loading up certain files when the EMBER_CLI_FASTBOOT environment variable is present

**Comment by dunnkers** [comment]
Good point! Thanks for opening the issue. I will look into [excluding node incompatible third party libraries](https://ember-fastboot.com/docs/addon-author-guide#third-party-dependencies) when the `EMBER_CLI_FASTBOOT` environment variable is true.

**Comment by dunnkers** [comment]
The `EMBER_CLI_FASTBOOT` environment variable seems to have been deprecated in Fastboot 1.x versions, investigating on using [this strategy](https://github.com/ember-fastboot/ember-cli-fastboot#build-hooks-for-fastboot) now.

**Comment by dunnkers** [comment]
Changes will be tracked in [fastboot-support](https://github.com/dunnkers/ember-polymer/tree/fastboot-support) branch.

Using following resources as guidelines:
- https://github.com/ember-fastboot/ember-cli-fastboot#disabling-incompatible-dependencies
- https://github.com/ember-fastboot/ember-cli-fastboot/issues/413
- https://gist.github.com/kratiahuja/d22de0fb1660cf0ef58f07a6bcbf1a1c#using-processenvember_cli_fastboot-to-run-import-in-browser-build.
